package.json

```diff
@@ -30,7 +30,10 @@
     "url": "https://github.com/jstransformers/jstransformer-lodash/blob/master/LICENSE.md"
   },
   "dependencies": {
-    "lodash": "^3.6.0"
+    "lodash.assign": "^3.2.0",
```

Comment:
`^` will pull down the latest `v3` so you shouldn't ever get `3.0` and since lodash follows semver your code will likely be fine.